### PR TITLE
CTX-5384: Added Utils method for formatting bytes to human readable format.

### PR DESCRIPTION
**Pull request by jovicazaric**

### CODE REVIEW

File: coretex/utils/number.py [hunk 1]
```diff
@@ -39,3 +39,25 @@ def mathematicalRound(value: float, decimalPlaces: int) -> float:
...
+def formatBytes(value: int, precision = 2):
+    """
+    Format a size in bytes into a human-readable format.
+
+    Parameters:
+        value (int): The size in bytes.
+        precision (int): Number of decimal places (default is 2).
+
+    Returns:
+        str: Formatted string with size and appropriate unit.
```
Comment (dule1322) [inline coretex/utils/number.py:53]: Use the same formatting as we do for documentation of other functions

File: coretex/utils/number.py [hunk 2]
```diff
@@ -39,3 +39,25 @@ def mathematicalRound(value: float, decimalPlaces: int) -> float:
     places = Decimal(10) ** -decimalPlaces
 
     return float(decimal.quantize(places, rounding = ROUND_HALF_UP))
+
+
+def formatBytes(value: int, precision = 2):
```
Comment (dule1322) [inline coretex/utils/number.py:44]: Explicitly annotate type of `precision` and return type of the function.

File: coretex/utils/number.py [hunk 3]
```diff
@@ -39,3 +39,25 @@ def mathematicalRound(value: float, decimalPlaces: int) -> float:
...
+
+    Parameters:
+        value (int): The size in bytes.
+        precision (int): Number of decimal places (default is 2).
+
+    Returns:
+        str: Formatted string with size and appropriate unit.
+    """
+
+    suffixes = ['B', 'KB', 'MB', 'GB', 'TB', 'PB', 'EB', 'ZB', 'YB']
```
Comment (dule1322) [inline coretex/utils/number.py:56]: Use double quotes for strings.

File: coretex/utils/number.py (hunk 1 above)
Comment (jovicazaric) [reply coretex/utils/number.py]: Updated

File: coretex/utils/number.py (hunk 2 above)
Comment (jovicazaric) [reply coretex/utils/number.py]: Added.

File: coretex/utils/number.py (hunk 3 above)
Comment (jovicazaric) [reply coretex/utils/number.py]: Updated

**Pull request merged**